src/views/ops/Template/components/Variable.vue

```diff
@@ -44,6 +44,10 @@ export default {
     value: {
       type: [Array],
       default: () => []
+    },
+    disableEdit: {
+      type: Boolean,
+      default: false
     }
   },
   data() {
```

Comment:
It seems there is no code provided to analyze the differences between two versions. Please upload or share the specific code you would like me to compare and analyze.